app/containers/WeatherContainer/index.js

```diff
@@ -0,0 +1,263 @@
+/**
+ *
+ * WeatherContainer
+ *
+ */
+
+import React, { useEffect, memo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, FormattedMessage as T } from 'react-intl';
+import Clickable from '@components/Clickable';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { useInjectSaga } from '@utils/injectSaga';
+import { selectWeatherContainer, selectCityData, selectCityError, selectCityName } from './selectors';
+import { weatherContainerCreators } from './reducer';
+import saga from './saga';
+import { Card, Skeleton, Input } from 'antd';
+import styled from 'styled-components';
+import debounce from 'lodash/debounce';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
as a concvntion we order the imports as follows

// third party libraries
// alias imports
// relative imports